Sources/Interaction/Manipulators/MouseRangeManipulator/index.js

```diff
@@ -28,7 +28,7 @@ function vtkMouseRangeManipulator(publicAPI, model) {
 
     const difference = value - listener.min;
     const stepsToDifference = Math.round(difference / listener.step);
-    value = listener.min + listener.step * stepsToDifference;
+    value = listener.min + (listener.step + listener.scale) * stepsToDifference;
```

Comment:
I don't think this works:
- if you use an integer scale, then you will always skip some values, even if you try to go slowly.
  - Ex: you want to go from slice 1 to slice 2 faster, not go from slice 1 to slice 3 at the same speed as you went from slice 1 to slice 2 while skipping slice 2.
- if you use a floating-point scale, then your `value` doesn't follow the stepping mechanism.
  - Ex: you could be at slice 1.5

I think applying the scale on this line is similar to just changing the `step` value. Instead, I'd recommend trying updating the delta value directly above (input from the mouse event that we want to accelerate or decelerate):
```js
const oldValue = listener.getValue()
const newDelta = delta * listener.scale + incrementalDelta.get(listener)
let value = oldValue + newDelta
```

If this works, you'd want to make sure to set `incrementalDelta` to `newDelta` if you haven't changed value ([on line 42](https://github.com/Kitware/vtk-js/pull/1900/commits/89453ba478ea295ddbc29cc7a15918ce8c16f09c#diff-e5dc7da166e6ad75f440a37272ace4a141e12d8495d59ddb22cb6d174d5d3b99R42))

Reply:
Got it, I'll take those suggestions and then validate it and report back! I truly did steal 100% of my swag from Jonas didn't I 😂 . What can I say... _Jonas, Ich bin du_